pkg/skaffold/docker/docker_init.go

```diff
@@ -0,0 +1,89 @@
+/*
+Copyright 2019 The Skaffold Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package docker
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/constants"
+	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/schema/latest"
+	"github.com/moby/buildkit/frontend/dockerfile/command"
+	"github.com/moby/buildkit/frontend/dockerfile/parser"
+	"github.com/sirupsen/logrus"
+)
+
+// Dockerfile is the path to a dockerfile. Implements the InitBuilder interface.
+type Dockerfile string
```

Comment:
I wonder if there is a better name, since `Dockerfile` is very general. I see in the follow-up PR the name is `Config` for the Jib InitBuilder, so that's a little bit inconsistent too.

Reply:
We could call this a `Config` as well (so there's a `jib.Config` and a `docker.Config`), the reason I called this one `Dockerfile` is because it's an alias for `string` that just holds a path to a Dockerfile.